src/main/scala/se/reciba/api/recibase/MealDefinitions.scala

```diff
@@ -513,7 +513,7 @@ object MealDefinitions {
       MealStub("Tuna Pasta", Set(Tag.Pescatarian, Tag.Quick, Tag.LowEffort)),
       MealStub(
         "Tuna steaks with salsa verde",
-        Set(Tag.Pescatarian),
+        Set(Tag.Pescatarian, Tag.Stephani, Tag.StephaniIsh),
```

Comment:
Ah wait, it's supposed to be Stephani _or_ Stephani-ish, depending on whether the meal perfectly matched dietary requirements vs if it could be non-destructively modified to meet requirements